### PR TITLE
Use eclipse-velocitas/cli latest release, fixup lfs

### PR DESCRIPTION
**Pull request by doosuu**
Clean up this repo by removing all LFS files by switching to the latest release of eclipse-velocitas/cli

### CODE REVIEW

File: common/install-velocitas-cli.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Copyright (c) 2022 Robert Bosch GmbH and Microsoft Corporation
```
Comment (MP91) [inline common/install-velocitas-cli.sh]:
```suggestion
# Copyright (c) 2022 Robert Bosch GmbH
```

File: common/install-velocitas-cli.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+CLI_INSTALL_PATH=/usr/bin/velocitas
+
+CLI_DOWNLOAD_URL="https://github.com/eclipse-velocitas/cli/releases/latest/download/${CLI_ASSET_NAME}"
+
+curl -L ${CLI_DOWNLOAD_URL} -o "${CLI_INSTALL_PATH}"
+chmod +x "${CLI_INSTALL_PATH}"
+
+CLI_VERSION=$( "${CLI_INSTALL_PATH}" --version )
+
+echo "CLI version: ${CLI_VERSION}"
```
Comment (MP91) [inline common/install-velocitas-cli.sh]: Optional
```suggestion
echo "Installed CLI version: ${CLI_VERSION}" 
```

File: common/install-velocitas-cli.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+echo "#######################################################"
+echo "### Install Velocitas CLI                           ###"
+echo "#######################################################"
+
+CLI_ASSET_NAME=velocitas-linux-$(echo $1 | sed 's/amd64/x64/g')
+CLI_INSTALL_PATH=/usr/bin/velocitas
```
Comment (MP91) [inline common/install-velocitas-cli.sh:21]: do we need to export this as system env for later usage?

**Review by MP91 — CHANGES_REQUESTED** [review]
just some minor suggestions

File: common/install-velocitas-cli.sh (hunk 3 above)
Comment (doosuu) [reply common/install-velocitas-cli.sh:21]: Not really needed since its available on the PATH since it is installed into '/usr/bin'

Comment (MP91) [reply common/install-velocitas-cli.sh:21]: true

**Review by MP91 — APPROVED** [review]
LGTM

**Pull request merged**